libraries/AP_Compass/AP_Compass_Calibration.cpp

```diff
@@ -1,7 +1,6 @@
 #include <AP_HAL/AP_HAL.h>
 #include <AP_Notify/AP_Notify.h>
 #include <AP_GPS/AP_GPS.h>
-#include <GCS_MAVLink/GCS.h>
```

Comment:
We need this for doing send_text, again it's transitive right now, this doesn't actually reduce the scope of work being recompiled, and makes it a future pain point.

Reply:
removed